dashboard/modules/state/state_head.py

```diff
@@ -347,6 +371,40 @@ async def list_cluster_events(
         record_extra_usage_tag(TagKey.CORE_STATE_API_LIST_CLUSTER_EVENTS, "1")
         return await self._handle_list_api(self._state_api.list_cluster_events, req)
 
+    @routes.get("/api/v1/cluster_events")
```

Comment:
hmm let's just remove v0 impl. we don't need to care the backward compatibility now

Reply:
Do we also need to remove the test cases? Since test_event is based on the v0 api

Reply:
You should rewrite them to be compatible to new API